### DISCUSSION TITLE
[in long box] Обращение к набору полей через код.

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#164] [Есть идея, а что если...]
![image](https://user-images.githubusercontent.com/103171127/176172764-ff162c66-9b54-41ae-8be6-939c6b8d14fb.png)
Просьба разработать обращение к набору полей через коды-действий... 
Это было бы очень удобным механизмом, нежели скрытие через поле в хедере по условию в Форматировании таблицы...

**Comment by ttm-alexeypolunin** [comment]
Это нельзя сделать тк набор в браузере у пользователя, а код действия на сервере и доступа к браузеру у него нет.

Если набор сохраняется для всех пользователей [дискетка] то он хранится в `Список таблиц` — `fields_sets`, а для таблиц внутри циклов `Версии расчетных таблиц` — `fields_sets`.

Вы можете изменять эти поля кодом, но изменение в браузере у пользователя произойдет только когда он снова зайдет в `глаз` и выберет этот набор. Единственное, что вы можете сделать с наборами с сервера — это сбросить вид таблицы у пользователя к виду по умолчанию в момент, когда он откроет таблицу — параметр [Актуальность настроек полей (fields_actuality)](https://ru.docs.totum.online/tables#table-settings-fields_actuality).

Какие-либо изменения в этой механике возможны если мы получим фондирование свыше $1М, до этого момента никаких ресурсов на изменение настроек видимости полей нет.
